### PR TITLE
Draft PR for importing a save_file

### PR DESCRIPTION
**Pull request by Vanderscycle**
Motivation:
Created this PR to learn about `initContainers`  and also to import current servers safe files in case I need to rebuild the cluster. 

Status:
PR is a draft at the moment as the charts builds, but the init pod that cp the save file doesn't and I need a bit of help/time to get to completion.

The file `test.yaml` will not be included once working, currently as a reproductible values file.

Related to discussion #32  
Any help is appreciated.

### CODE REVIEW

**Comment by Vanderscycle** [comment]
Is there a way that I can make this PR a draft?

File: charts/factorio-server-charts/templates/deployment.yaml [hunk 1]
```diff
@@ -102,7 +102,23 @@ spec:
...
+        - name: load-save-file
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy:  {{ .Values.image.pullPolicy }}
+          command:
+              - /bin/bash
+              - -ec
+              - |
+                ls -al 
+                mkdir /factorio/saves
+                echo "Copying saveFile from {{ .Values.factorioServer.local_save_file_name }} to /factorio/saves/{{ .Values.factorioServer.save_name }}.zip"
```
Comment (SQLJames) [inline charts/factorio-server-charts/templates/deployment.yaml:116]: could we just use the same save file name for both? I believe the save file name is important for loading the game on restart, you might end up generating a new save if not named properly. (i believe)

**Comment by SQLJames** [comment]
> Is there a way that I can make this PR a draft?

done.

**Comment by SQLJames** [comment]
Other than my comment this looks like a solid foundation. If you want to bounce ideas around or get stuck on let me know :)

**Comment by Vanderscycle** [comment]
I am thinking roadmap wise regarding discussion #32 that we can figure out how to import a local save file and a subsequent PR could focus on i.porting from a cloud provider: s3, gDrive, etc.

**Comment by SQLJames** [comment]
#40 closes this issue

**Pull request closed without merging**